CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+Unreleased changes:
+* -0 is now a valid value for Double and Float.
```

Comment:
Could you please split this up into a changelog in `validity` and one in `genvalidity`? One per package is the standard way of doing things.